lib/active_record_shards/connection_switcher.rb

```diff
@@ -171,6 +169,17 @@ def config_for_env
     end
     alias_method :check_config_for_env, :config_for_env
 
+    def all_shard_names_are_integers?
+      @all_shard_names_are_integers ||= {}
+      @all_shard_names_are_integers[shard_env] ||= begin
```

Comment:
Let’s continue the pattern from `#config_for_env`.
```suggestion
      @_ars_all_shard_names_are_integers ||= {}
      @_ars_all_shard_names_are_integers[shard_env] ||= begin
```

Reply:
I think I tried that and rubocop yelled at me, but let me check.